apps/nextjs/tests-e2e/tests/aila-chat/full-romans.mobile.test.ts

```diff
@@ -15,8 +15,8 @@ import {
 // --------
 // CHANGE "replay" TO "record" TO RECORD A NEW FIXTURE
 // --------
-// const FIXTURE_MODE = "record" as FixtureMode;
-const FIXTURE_MODE = "replay" as FixtureMode;
+const FIXTURE_MODE = "record" as FixtureMode;
```

Comment:
This test is broken for me on local dev. It tries to connect using the "https://" scheme